### PR TITLE
Show user immediately if selection correct or not function completed

### PR DESCRIPTION
**Pull request by AliDemircix**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -1,106 +1,219 @@
 'use strict';
 
 /* Program Data
-
```
Comment (killoncio) [inline src/data.js]: Same, data.js with unintended modifications

File: src/pages/questionPage.js [hunk 2]
```diff
@@ -23,13 +23,25 @@ export const initQuestionPage = (userInterface) => {
...
 // check answers if correct or not 
+let isSelectedOneAnswer= false;
 const getAnswer = (e)=> {
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+if(!isSelectedOneAnswer){
   const answer = e.target.innerText;
   currentQuestion.selected=answer.charAt(0)
-  currentQuestion.selected===currentQuestion.correct ? console.log('selected correct answer') : console.log('selected wrong answer')
+  showAnswerIsCorrect(currentQuestion.selected, currentQuestion.correct,e.target)
+  isSelectedOneAnswer=true;
```
Comment (killoncio) [inline src/pages/questionPage.js]: This is part of the other PR, isn't it?

File: src/pages/questionPage.js [hunk 3]
```diff
@@ -23,13 +23,25 @@ export const initQuestionPage = (userInterface) => {
...
 }
+}
+// Show user if user selection correct or not
+const showAnswerIsCorrect =(selectedAnswer,correctAnswer,target)=> {
```
Comment (killoncio) [inline src/pages/questionPage.js]: the `=` before `(selectedAnswer` is a typo?

File: src/pages/questionPage.js [hunk 4]
```diff
@@ -23,13 +23,25 @@ export const initQuestionPage = (userInterface) => {
...
 };
 // check answers if correct or not 
+let isSelectedOneAnswer= false;
 const getAnswer = (e)=> {
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+if(!isSelectedOneAnswer){
   const answer = e.target.innerText;
   currentQuestion.selected=answer.charAt(0)
-  currentQuestion.selected===currentQuestion.correct ? console.log('selected correct answer') : console.log('selected wrong answer')
+  showAnswerIsCorrect(currentQuestion.selected, currentQuestion.correct,e.target)
```
Comment (killoncio) [inline src/pages/questionPage.js]: You don't need to pass `currentQuestion.selected` and `currentQuestion.correct`, you can just pass `currentQuestion.selected === currentQuestion.correct`, or create a variable called eg `isAnswerCorrect` and store there `currentQuestion.selected === currentQuestion.correct`, and then pass that variable as argument to `showAnswerIsCorrect`

File: src/pages/questionPage.js (hunk 2 above)
Comment (AliDemircix) [reply src/pages/questionPage.js]: Yes I continued on first task.

File: src/pages/questionPage.js (hunk 3 above)
Comment (AliDemircix) [reply src/pages/questionPage.js]: It is for defining arrow function const arrowFunc = () => { }

Comment (killoncio) [reply src/pages/questionPage.js]: haha, sorry, I normally leave a space between them, but it's good as it is

File: src/pages/questionPage.js [hunk 5]
```diff
@@ -23,13 +23,27 @@ export const initQuestionPage = (userInterface) => {
...
+  currentQuestion.selected = answer.charAt(0)
+  const isAnswerCorrect = currentQuestion.selected===currentQuestion.correct
+  showAnswerIsCorrect(isAnswerCorrect,e.target)
+  isSelectedOneAnswer = true;
+}
 }
+// Show user if user selection correct or not
+const showAnswerIsCorrect = (isAnswerCorrect,target) => {
+  isAnswerCorrect ? setBackgroundColor('green',target) : setBackgroundColor('red',target)
+  }
```
Comment (killoncio) [inline src/pages/questionPage.js]: Can you please make sure the indentations are correct? In this line, for example, there should be no indentation. 

L 42,43 and 44 is also not correct.

You are also missing `;` in many lines, like 32,33, 34,40. Please, make sure they all have it.

For the rest I think it's good!!

Comment (AliDemircix) [reply src/pages/questionPage.js]: I think my prettier is not working for this project. I corrected them manually and now sending it.

**Review by killoncio — APPROVED** [review]
Good job!

**Pull request merged**